code/controllers/subsystem/vote.dm

```diff
@@ -341,7 +341,7 @@ SUBSYSTEM_DEF(vote)
 /datum/controller/subsystem/vote/proc/remove_action_buttons()
 	for(var/v in generated_actions)
 		var/datum/action/innate/vote/V = v
-		if(!QDELETED(V))
+		if(!QDELETED(V?.owner))

```

Comment:
This changes the behavior slightly.
It will check if `V` exists, if `V.owner` exists and `V.owner.gc_destroyed` is not true.
Before it checked for `V` and `V.gc_destroyed`.
Is this how you want it to behave?